### PR TITLE
Test PR CR

### PR DESCRIPTION
**Pull request by jakeschievink**

### CODE REVIEW

File: twin/incoming-voice-dial.php [hunk 1]
```diff
@@ -23,6 +24,9 @@
 
 // is the call still active?
 if ($call_status != 'completed') {
+    if ($digit == 0) {
```
Comment (sharett) [inline twin/incoming-voice-dial.php]: add a comment to explain?

File: twin/incoming-voice.php [hunk 2]
```diff
@@ -43,6 +43,8 @@
 	);
 }
 
+$gather->say("or press 0 for voicemail");
```
Comment (sharett) [inline twin/incoming-voice.php]: Avoid hard coding text - add to config.php
Use the same voice as the other prompts

File: twin/voicemail.php [hunk 3]
```diff
@@ -21,9 +21,11 @@
 sms_loadLanguage($language_id, $language, $error);
 
 // no one available to answer
+if ($language_id != 0) {
 $response->say($language['voicemail'],
```
Comment (sharett) [inline twin/voicemail.php]: not indented?

**Review by sharett — CHANGES_REQUESTED** [review]
things to address

File: twin/incoming-voice-dial.php (hunk 1 above)
Comment (jakeschievink) [reply twin/incoming-voice-dial.php]: done

File: twin/voicemail.php (hunk 3 above)
Comment (jakeschievink) [reply twin/voicemail.php]: done

File: twin/incoming-voice.php (hunk 2 above)
Comment (jakeschievink) [reply twin/incoming-voice.php]: done

**Review by sharett — APPROVED** [review]
Looks good!

**Pull request merged**